test/unit/interpolation.test.js

```diff
@@ -80,6 +80,25 @@ describe('component interpolation', () => {
       })
     })
 
+    describe('empty text node between components', () => {
+      it('should be interpolated', done => {
```

Comment:
Since this test case describes `'empty text node between components'`,

```suggestion
      it('should NOT be interpolated', done => {
```